lib/r2.rb

```diff
@@ -110,7 +110,7 @@ def minimize(css)
       return '' unless css
 
       css.gsub(/\/\*[\s\S]+?\*\//, '').   # comments
-         gsub(/[\n\r]/, '').              # line breaks and carriage returns
+         gsub(/[\n\r]/, ' ').             # line breaks and carriage returns
```

Comment:
Re minimization - IMO it should be a job of another post-processor/library. It's better if R2rb has a single focus. Plus, it would make for easier tests.
Anyway, I'll add your `/[\n\r]+/` suggestion and push a patch right away.